sqlest/src/test/scala/sqlest/sql/DB2StatementBuilderSpec.scala

```diff
@@ -235,6 +235,19 @@ class DB2StatementBuilderSpec extends BaseStatementBuilderSpec {
     )
   }
 
+  it should "produce the right SQL for nullable literal columns" in {
+    sql {
+      select(TableThree.col3, TableThree.col4, Option.empty[String].constant.as("testOption"))
```

Comment:
I think that the first one is probably one implicit conversion too far away from compiling, I think this is okay as this is the actual case you wanted to solve - allowing a literal `null` in your query